README.md

```diff
@@ -101,19 +101,31 @@ common     | authorized\_keys          | updates the authorized\_keys file for t
 rails      | appyml                    |
 monit      | monit                     | All tasks in the monit role have been tagged 'monit'
 
-### `subspace vars <environment> [--edit] [--create]`
+### `subspace secrets <environment> [--edit] [--create]`
 
-Manage environment variables on different platforms.  The default action is simply to show the vars defined for an environment.  Pass --edit to edit them in the system editor.
+The `secrets` command will manage encrypted secrets for different environments.  The default action is simply to show the secrets defined for an environment.  Pass --edit to edit them in the system editor (vim, etc).
 
-The new system uses a file in `config/provision/templates/application.yml.template` that contains environment variables for all environments.  The configuration that is not secret is visible and version controlled, while the secrets are stored in the vault files for their environments. The default file created by `subspace init`  looks like this:
+This uses `ansible-vault` under the hood and requires a vault password file.  You will need to get the `.vault_pass` from from a teammate out of band, and put it into `config/provision/.vault_pass`
```

Comment:
we should specify what "out of band" is, like use secrets.10fw.net or something. writing this here so i don't forget to make this change